src/server.rs

```diff
@@ -182,7 +216,7 @@ fn read_opt(
     // Validate::from_request extracts the token from the Authorization header, validates it
     // against FxA and the method, and either returns OK or an error. We need to reraise it to the
     // handler.
-    slog_info!(logger.log, "Handling Read");
+    slog_debug!(logger.log, "Handling Read"; "user_id"=>&user_id, "device_id"=>&device_id);
```

Comment:
nit: (here and elsewhere) unfortunately cargo fmt doesn't touch this macro, regardless we usually wrap the =>'s  in spaces